I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/quiz_gen.rpy", line 474, in script
    $ process = subprocess.Popen([python_path, py_path, quiz_title, notes], creationflags=subprocess.CREATE_NO_WINDOW)
  File "game/quiz_gen.rpy", line 474, in <module>
    $ process = subprocess.Popen([python_path, py_path, quiz_title, notes], creationflags=subprocess.CREATE_NO_WINDOW)
TypeError: expected str, bytes or os.PathLike object, not NoneType

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/quiz_gen.rpy", line 474, in script
    $ process = subprocess.Popen([python_path, py_path, quiz_title, notes], creationflags=subprocess.CREATE_NO_WINDOW)
  File "C:\Users\tagle\OneDrive\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\tagle\OneDrive\Desktop\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/quiz_gen.rpy", line 474, in <module>
    $ process = subprocess.Popen([python_path, py_path, quiz_title, notes], creationflags=subprocess.CREATE_NO_WINDOW)
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/subprocess.py", line 951, in __init__
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/subprocess.py", line 1360, in _execute_child
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/subprocess.py", line 565, in list2cmdline
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/os.py", line 822, in fsdecode
TypeError: expected str, bytes or os.PathLike object, not NoneType

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Kodigo 1.0
Thu Mar 28 17:44:35 2024
